build/sync-translated-content.js

```diff
@@ -95,6 +95,14 @@ function resolve(slug) {
   return slug;
 }
 
+function mdOrHtmlExists(filePath) {
+  const { dir, name } = path.parse(filePath);
+  return (
+    fs.existsSync(path.join(dir, name) + ".md") ||
+    fs.existsSync(path.join(dir, name) + ".html")
+  );
```

Comment:
```suggestion
  const dir = path.dirname(filePath);
  return (
    fs.existsSync(path.join(dir, MARKDOWN_FILENAME) ||
    fs.existsSync(path.join(dir, HTML_FILENAME)
  );
```
Really not sure which one is better. Your choice.

Reply:
Ah, I didn't see those constants - I like this